### ISSUE TITLE
[SVG] Broken attributes, Wrong documentation.

### ISSUE THREAD

**Issue by juancarlospaco** [brython-dev/brython#71]
```python
Brython 3.0.0 on Netscape 5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/38.0.2125.111 Chrome/38.0.2125.111 Safari/537.36
>>> from browser import document, svg
>>> document["main"] <= svg.svg(id="test")
>>> document["test"] <= svg.text("testing", stroke="red", stroke_width="9")
```

Returns Broken attributes, invalid as SVG XML:
```xml
<text stroke="red" stroke_width="9">testing</text>
```
(stroke_width its not working)

This should be:
```xml
<text stroke="red" stroke-width="9">testing</text>
```

- Brython must replace the Underscores on attributes with Hypens to be Valid SVG on the generated SVG.
- Documentation must be modified to reflect that, and mention the conversion of Underscores to Hypens.

:dizzy:

**Comment by PierreQuentel** [comment]
Thanks for the report. I think this is fixed in commit 0f2a1fc69d86c525ed616895975ea727af9e9b2c (only the French version of the documentation is updated)

If you confirm I will close this issue

**Comment by juancarlospaco** [comment]
:+1:
